assets/i18n/i18n_en.properties

```diff
@@ -2,4 +2,7 @@ touchToBegin=Touch to begin
 gameTitle=Dark Matter
 startGame=Start Game
 quitGame=Quit Game
-highscore=Highscore
+highscore=Highscore: {0}
+areYouSure=Are you sure you want to quite the game?
```

Comment:
You do not have to fill the bundles twice. Either keep English texts in the default bundle, or empty values in the default bundle and English texts here.